actiontext/lib/action_text/fixture_set.rb

```diff
@@ -1,7 +1,57 @@
 # frozen_string_literal: true
 
 module ActionText
+  # \Fixtures are a way of organizing data that you want to test against; in
+  # short, sample data.
+  #
+  # To learn more about fixtures, read the ActiveRecord::FixtureSet
+  # documentation.
+  #
+  # === YAML
+  #
+  # Like other Active Record-backed models, ActionText::RichText records inherit
+  # from ActiveRecord::Base instances and therefore can be populated by
+  # fixtures.
+  #
+  # Consider a hypothetical <tt>Article</tt> model class, its related
+  # fixture data, as well as fixture data for related ActionText::RichText
+  # records:
+  #
+  #   # app/models/article.rb
+  #   class Article < ApplicationRecord
+  #     has_rich_text :content
+  #   end
+  #
+  #   # tests/fixtures/articles.yml
+  #   first:
+  #     title: An Article
+  #
+  #   # tests/fixtures/action_text/rich_texts.yml
+  #   first_content:
+  #     record: first (Article)
+  #     name: content
+  #     body: <div>Hello, world.</div>
+  #
+  # When processed, Active Record will insert database records for each fixture
+  # entry and will ensure the Action Text relationship is intact.
   class FixtureSet
+    # Fixtures support Action Text attachments as part of their <tt>body</tt>
+    # HTML.
+    #
+    # === Examples
+    #
+    # For example, consider a second <tt>Article</tt> record that mentions the
+    # first as part of its <tt>content</tt> HTML:
+    #
+    #   # tests/fixtures/articles.yml
+    #   second:
+    #     title: Another Article
+    #
+    #   # tests/fixtures/action_text/rich_texts.yml
+    #   second_content:
+    #     record: second (Article)
+    #     name: content
+    #     body: <div>Hello, <%= ActionText::FixtureSet.attachment("artcles", :first) %></div>
```

Comment:
@georgeclaghorn after having a chance to reflect, how do we feel about the separation of `ActionText::FixtureSet` from `ActiveRecord::FixtureSet`?

Reply:
I think it's fine and good. Active Record shouldn't know about the setup required for fixtures in Action Text.